demo/admin/src/products/ProductForm.tsx

```diff
@@ -150,54 +179,119 @@ export function ProductForm({ id }: FormProps): React.ReactElement {
-                            label={<FormattedMessage id="product.description" defaultMessage="Description" />}
-                        />
-                        <SelectField name="type" label="Type" required fullWidth>
-                            <MenuItem value="Cap">Cap</MenuItem>
-                            <MenuItem value="Shirt">Shirt</MenuItem>
-                            <MenuItem value="Tie">Tie</MenuItem>
-                        </SelectField>
-                        <SelectField name="additionalTypes" label="Additional Type" required fullWidth multiple>
-                            <MenuItem value="Cap">Cap</MenuItem>
-                            <MenuItem value="Shirt">Shirt</MenuItem>
-                            <MenuItem value="Tie">Tie</MenuItem>
-                        </SelectField>
-                        <Field
-                            fullWidth
-                            name="category"
-                            label="Category"
-                            component={FinalFormSelect}
-                            {...categorySelectAsyncProps}
-                            getOptionLabel={(option: GQLProductCategorySelectFragment) => option.title}
-                        />
-                        <Field
-                            fullWidth
-                            name="tags"
-                            label="Tags"
-                            component={FinalFormSelect}
-                            multiple
-                            {...tagsSelectAsyncProps}
-                            getOptionLabel={(option: GQLProductTagsSelectFragment) => option.title}
-                        />
-                        <CheckboxField name="inStock" label={<FormattedMessage id="product.inStock" defaultMessage="In stock" />} fullWidth />
-                        <Field name="image" isEqual={isEqual}>
-                            {createFinalFormBlock(rootBlocks.image)}
-                        </Field>
-                    </MainContent>
-                </>
-            )}
+            {({ values, form }) => {
+                return (
+                    <>
+                        {saveConflict.dialogs}
+                        <MainContent>
+                            <TextField required fullWidth name="title" label={<FormattedMessage id="product.title" defaultMessage="Title" />} />
+                            <TextField required fullWidth name="slug" label={<FormattedMessage id="product.slug" defaultMessage="Slug" />} />
+                            <TextAreaField
+                                required
+                                fullWidth
+                                name="description"
+                                label={<FormattedMessage id="product.description" defaultMessage="Description" />}
+                            />
+                            <>
+                                <AsyncSelectField
+                                    name="manufacturerFilter"
```

Comment:
Naming: How about `country`?

Reply:
I thought about that but decided to name it this way to prevent a misconception this could be a country-field of product-entity. Maybe manufacturerCountry? With using the Filter-suffix it's clear that's not part of the mutation.

Reply:
manufacturerCountry is fine 👍🏼